10_integrations/algolia_indexer.py

```diff
@@ -112,7 +112,7 @@
 # ## The actual function
 #
 # We want to trigger our crawler from our CI/CD pipeline, so we're serving it as a
-# [web endpoint](/docs/guide/webhooks#web_endpoint) that can be triggered by a `GET` request during deploy.
+# [web endpoint](/docs/guide/webhooks) that can be triggered by a `GET` request during deploy.
```

Comment:
We should change this to an absolute URL. That's the style we use elsewhere because it works better when the examples are viewed in editors. It means that we can't catch 404s as easily, but it also reduces what are IMO noisy build errors like this one from a changed fragment identifier.

Reply:
That makes sense. I see a few others so I'll leave it to be done in all one coherent PR.